packages/contracts/src/diamond/facets/StakingFormulasFacet.sol

```diff
@@ -2,15 +2,15 @@
 pragma solidity ^0.8.16;
 
 import {LibStakingFormulas} from "../libraries/LibStakingFormulas.sol";
-import {StakingShare} from "../../dollar/StakingShare.sol";
+import {StakingShareForDiamond} from "../token/StakingShareForDiamond.sol";
```

Comment:
I thought we were having a separate manager for the tokens

Reply:
IAccessControl resides inside the diamond and all tokens have the diamond address to check the access for minting and burning